src/e2eTest/utils/helpers/caseApi.ts

```diff
@@ -0,0 +1,80 @@
+// src/e2eTest/utils/helpers/caseApi.ts
+import Axios, {AxiosInstance, AxiosResponse} from 'axios';
+import {
+  getIdamAuthToken,
+  getServiceAuthToken
+} from '../helpers/idam-helpers/idam.helper';
+import {TestConfig} from 'config/test.config';
+import caseDataJson from '../../data/case.data.json';
+
+
+export interface CaseWithId {
+  id: string;
+  state: string;
+  ccd_case_number?: string;
+  case_reference?: string;
+}
+
+export interface CcdTokenResponse {
+  token: string;
+}
+
+export interface CcdV2Response {
+  id: string;
+  state: string;
+  data: any;
+}
+
+export class CaseApi {
+  constructor(
+    private readonly axios: AxiosInstance,
+  ) {
+  }
+
+  async createCase(caseType: string, appId: string): Promise<CaseWithId> {
```

Comment:
Another renaming suggestion, appId is not a familiar terminology in CCD, this would be better named as eventName.